ant_plus_sensor/ant_plus_logger.js

```diff
@@ -156,6 +187,21 @@ async function heartRateConnect(antPlus) {
     }
   });
 
+  const bicycleSpeedScanner = await bicycleSpeedConnect(antPlus);
+  bicycleSpeedScanner.on('speedData', (data) => {
+    // Store speed into global variable
+    speed = data.CalculatedSpeed;
+    if (startWheelRevolutions === null) {
+      startWheelRevolutions = data.CumulativeSpeedRevolutionCount;
```

Comment:
Just for my own understanding, startWheelRevolutions would ideally be 0 right?

Reply:
Yeah so I think `CumulativeSpeedRevolutionCount` is actually stored on the ant+ sensor and sent to the dongle with each message. This means that if the speed sensor is turned on (which it will do automatically when it detects wheel movement) it will start counting. This way we can set an appropriate offset.